Global drafts documentation (#6118)
mattermost/docs: source/configure/experimental-configuration-settings.rst

**False**: **(Default for self-hosted deployments)** All integration icons in the channel header display in the channel header.

+-------------------------------------------------------------------------------------------------------------------------------+
| This feature's ``config.json`` setting is ``"ExperimentalSettings.EnableAppBar": false`` with options ``true`` and ``false``. |
+-------------------------------------------------------------------------------------------------------------------------------+

Allow synchronized drafts
-------------------------

> This setting saves in-progress message drafts in the Drafts view via the channel sidebar, and synchronizes draft messages across all supported Mattermost clients.

Reviewer: This setting ~~saves in-progress message drafts in the Drafts view via the channel sidebar, and~~ synchronizes draft messages across all supported Mattermost clients.

> This setting saves in-progress message drafts in the Drafts view via the channel sidebar, and synchronizes draft messages across all supported Mattermost clients.

Reviewer 2: ```suggestion
This setting synchronizes draft messages across all supported Mattermost clients.
```

> **True**: **(Default)** Global drafts are enabled, synchronized across clients, and draft messages are accessible through the Drafts view. 

Reviewer: True: Message drafts are saved on the server and may be accessed from different clients. Users may still disable server synchronization of draft messages in Account Settings.

> **True**: **(Default)** Global drafts are enabled, synchronized across clients, and draft messages are accessible through the Drafts view. 

Author: ```suggestion
**True**: **(Default)** Message drafts are saved on the server and may be accessed from different clients. Users may still disable server synchronization of draft messages by going to **Settings > Advanced Settings**. 
```

> **False**: Global drafts are disabled.

Reviewer: False: Draft messages are only stored locally on each users device

> **False**: Global drafts are disabled.

Author: ```suggestion
**False**: Draft messages are only stored locally on each user's device.
```
